index.js

```diff
@@ -1,24 +1,46 @@
 'use strict';
 const UNITS = ['B', 'kB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
 
-module.exports = num => {
+module.exports = (num, options) => {
 	if (!Number.isFinite(num)) {
 		throw new TypeError(`Expected a finite number, got ${typeof num}: ${num}`);
 	}
 
+	options = Object.assign({}, options);
+
 	const neg = num < 0;
 
 	if (neg) {
 		num = -num;
 	}
 
 	if (num < 1) {
+		num = toLocaleString(num, options.locale);
```

Comment:
Very minor, but I'd declare `let numStr` earlier and then assign it here, making it clearer that `num` is always a number.

Reply:
I agree. Also changed line 24, where the result is always a number and not a `numStr`